### DISCUSSION TITLE
Responsive position of dropdown menu

### DISCUSSION THREAD

**Discussion by pjschreifels** [tailwindlabs/tailwindcss#3036] [Help] [answered]
**Reference**: Application UI → Elements → Dropdown "With minimal menu icon"

The default position of the dropdown menu is `origin-top-right right-0` for all breakpoints, making the menu right-justified. I'd like the menu to be left-justified on mobile, then go back to right-justified at larger breakpoints. Seems pretty straight forward to set `origin-top-left left-0 sm:origin-top-right sm:right-0`, but the menu stays left-justified regardless of browser width.

Am I doing something wrong?

**Comment by MartinHughes-BPC** [comment, marked as answer]
Hey @pjschreifels 

I think what's happening is that the sm:right-0 is being applied in addition to the left-0. What happens if you do sm:left-auto as well as sm:right-0?

Cheers,
Martin.

**Reply by pjschreifels** [reply]
That makes perfect sense and resolved the issue. Thanks!
